components/authz-service/server/v2/policy_test.go

```diff
@@ -2938,11 +3104,17 @@ type testSetup struct {
 func setupV2WithWriter(t *testing.T,
 	writer engine.V2pXWriter) testSetup {
 	return setupV2WithMigrationState(t, nil, writer, nil, nil, make(chan api_v2.Version, 1),
-		// if the MigrationStatus is set to "Success", that means we've migrated
-		// successfully to IAM v2 ("SuccessBeta1" is v2.1).
+		// Returning MigrationStatus of "Success" means we've migrated successfully to IAM v2
 		func(s storage.MigrationStatusProvider) error { return s.Success(context.Background()) })
 }
 
+func setupV2p1WithWriter(t *testing.T,
+	writer engine.V2pXWriter) testSetup {
+	return setupV2WithMigrationState(t, nil, writer, nil, nil, make(chan api_v2.Version, 1),
+		// Returning MigrationStatus of "SuccessBeta2.1" means we've migrated successfully to IAM v2.1
+		func(s storage.MigrationStatusProvider) error { return s.SuccessBeta1(context.Background()) })
+}
```

Comment:
Thanks, @srenatus, for the framework that made adding v2.1 just plug-n-play, as it were!